engine/execution/computation/computer/result_collector.go

```diff
@@ -246,6 +246,17 @@ func (collector *resultCollector) processTransactionResult(
 		Int64("time_spent_in_ms", timeSpent.Milliseconds()).
 		Logger()
 
+	if output.ComputationUsed > 0 {
+		// log the normalized time per computation
+		// if the computation estimation is correct the value should be 1.
+		// if the value is greater than 1, the computation estimation is too low; we are underestimating transaction complexity (and thus undercharging).
+		// if the value is less than 1, the computation estimation is too high; we are overestimating transaction complexity (and thus overcharging).
```

Comment:
Does it matter if the transaction failed? Which might have a much smaller time per computation?

Meaning we might not be overcharging, but simply because the transaction failed.

Reply:
we should be charging correctly even if the transaction fails. The tx still takes time before failing.